.hlint.yaml

```diff
@@ -0,0 +1,2 @@
+- ignore: {name: Redundant <$>, within: [Hydra.Contract.HeadTokens]}
+- ignore: {name: Redundant compare, within: [Hydra.PartySpec]}
```

Comment:
Should: use a local ignore pragma or address this

This should be possible, no?

Reply:
I don't know really, GHC just doesn't like it when I do the suggestions in the docs.